src/components/shared/CacheWarning.tsx

```
@@ -0,0 +1,125 @@
+      const now = Date.now();
+      const lastCheck = localStorage.getItem('lastCacheCheck');
+      const cacheWarningDismissed = localStorage.getItem('cacheWarningDismissed');
+      
+      // Show warning if:
+      // 1. Cache issues detected, or
+      // 2. Never checked before, or
+      // 3. Last check was more than 1 hour ago, or
+      // 4. User hasn't dismissed it in the last 24 hours
+      const shouldShow = hasCacheIssues ||
+                        !lastCheck || 
+                        (now - parseInt(lastCheck)) > 3600000 || // 1 hour
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)